bisect_ppx-ocamlbuild.opam

```diff
@@ -0,0 +1,24 @@
+name: "bisect_ppx-ocamlbuild"
```

Comment:
I actually have a TODO item to write a brief "Depending" section in the README, and then announce that together with an explanation of how the Lwt breakage process works (with the major releases and warnings, etc.). Ideally, more projects would adopt that... but at a minimum, more people would subscribe to the announcements issue.

Does having `~unknown` as a version not have the problem of interacting with upper bounds? If so, I think you should start that thread... or if you can come up with any other good alternative.